prow/cmd/phaino/README.md

```diff
@@ -2,13 +2,23 @@
 
 Run prowjobs on your local workstation with `phaino`.
 
+Plato believed that [ideas and forms] are the ultimate truth,
+whereas we only see the imperfect physical appearances of those idea.
+
+He linkens this in his [Allegory of the Cave] to someone living in a cave
+who can only see the shadows projected on the wall
+from objects passing in front of a fire. 
+
+[Phaino] is act of making those imperfect shadows appear.
+
 ## Usage
 
 Usage:
 ```console
 # Use a job from deck
 bazel run //prow/cmd/phaino -- $URL # or /path/to/prowjob.yaml
-bazel run //prow/cmd/mkpj -- --config=/your/config --job=foo | bazel run //prow/cmd/phaino
+# Use mkpj to create the job
+bazel run //prow/cmd/phaino -- <(bazel run //prow/cmd/mkpj -- --config-path=/your/config --job=foo)
```

Comment:
I've never seen this syntax before. Neat.

Reply:
No! That glimmer of "neat" is the path to the dark side... Bash-isms...